javacutil/src/main/java/org/checkerframework/javacutil/CollectionUtils.java

```diff
@@ -1,11 +1,24 @@
 package org.checkerframework.javacutil;
 
+import java.util.Collection;
 import java.util.LinkedHashMap;
 import java.util.Map;
 
 /** Utility methods related to Java Collections. */
 public class CollectionUtils {
 
+    // The class object of "java.util.Collections$UnmodifiableCollection"
```

Comment:
```suggestion
    /** The class object of "java.util.Collections$UnmodifiableCollection" */
```

Reply:
I guess this is why the misc task failed. Can you check, reformat, and commit?

Reply:
Thanks! The task seems to pass locally.